tree_rightview.cpp

```diff
@@ -0,0 +1,62 @@
+#include<iostream>
+#include<queue>
+using namespace std;
+
+struct node
+{
+	int data;
+	node* left;
+	node* right;	
+};
+
+node* insertNode(int data)
+{
+	node* temp=new node;
+	temp->data=data;
+	temp->left=NULL;
+	temp->right=NULL;
+	return temp;
+}
+
+void rightView(node* root)
+{
+	if(root==NULL)
+	return;
+	queue<node*> q;
+	q.push(root);
+	while(1)
+	{
+		int nodeCount=q.size();
+		if(nodeCount==0)
+		break;
+		while(nodeCount--)
+		{
+			node* root=q.front();
+			q.pop();
+			if(nodeCount==0)
+			{
+				printf("%d ",root->data);
```

Comment:
printf -> cout

Reply:
cout is slow compared to printf (same as #include<bits/stdc++.h>)

Reply:
Code is already indented with one tab after each opening curly brace

Reply:
@ramini1996 You're really worried about speed? When properly used, cout can be as fast, even faster than printf. Here are some sources: (https://stackoverflow.com/a/33273370), (https://stackoverflow.com/a/12045514), (https://stackoverflow.com/a/896670). Since you are only printing to a console, the amount of efficiency your are striving for is premature.

> Premature optimization is the root of all evil - Donald Knuth

Reply:
As for your second comment, if you have a control statement, you should always indent what is inside for readability. It is confusing what you mean with the `bits/stdc++.h` thing.

Reply:
Updated it